lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -1,3 +1,4 @@
+require 'active_model_serializers/json_api_objects'
```

Comment:
It makes more sense there. It's AMS's representation of JSON API objects, not (necessarily) a sub-namespace of Serializer::Adapter::JsonAPI

And since we're building new stuff over there...